bridge-cli/src/omni_connector_command.rs

```diff
@@ -116,6 +116,21 @@ pub enum OmniConnectorSubCommand {
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Finalize a transfer on NEAR using fast transfer")]
```

Comment:
We need to update the README for every new command in the CLI.
I would recommend automating this process and using the crate https://github.com/ConnorGray/clap-markdown

Reply:
Btw, I think it can be integrated and fixed in a separate PR